cmd/main.go

```diff
@@ -2,29 +2,12 @@ package main
 
 import (
 	"fmt"
-	"io/ioutil"
-	"log"
-	"net/http"
 	"os"
+	"weather"
 )
 
 func main() {
 
-	token := os.Getenv("WEATHERAPP_TOKEN")
-	if len(token) == 0 {
-		fmt.Printf("please set a weatherapp token\n")
-		os.Exit(2)
-	}
-
-	//call open weather map and get something
-	resp, err := http.Get(fmt.Sprintf("https://api.openweathermap.org/data/2.5/weather?q=london&appid=%s", token))
-	if err != nil {
-		log.Fatal(err)
-	}
-	read_all, err := ioutil.ReadAll(resp.Body)
-
-	if err != nil {
-		log.Fatal(err)
-	}
-	fmt.Printf("%s", read_all)
+	resp := weather.RunCLI(os.Args)
+	fmt.Print(resp)
```

Comment:
We probably don't even need `fmt.Print`, do we? Since `RunCLI` does everything else, I think it's all right for it to do the printing, too.